opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/config/encryption/DatasafeConfig.java

```diff
@@ -102,6 +110,24 @@ public FintechConsentSpecSecureStorage fintechUserDatasafeServices(
         );
     }
 
+    @Bean
+    @ConfigurationProperties(prefix = "facade.datasafe")
+    public EncryptionConfig encryptionConfig() {
+        return EncryptionConfig.builder().build();
+    }
+
+    private EncryptionConfig buildEncryptionConfig() {
+        return datasafeConfigRepository.findAll().stream()
+                       .findFirst()
+                       .map(e -> mapToEncryptionConfig(e.getConfig()))
+                       .orElseGet(this::encryptionConfig);
+    }
+
+    @SneakyThrows
+    private EncryptionConfig mapToEncryptionConfig(byte[] data) {
+        return mapper.readValue(new String(data), EncryptionConfig.class);
```

Comment:
ObjectMapper can read directly from bytes, no need to do `new String(data)`

Reply:
Done, thanks